doc/en/user/source/data/app-schema/mapping-file.rst

```diff
@@ -278,6 +278,7 @@ E.g.::
 
 Note that the GML encoding rules require that complex types are never the direct property of another complex type; they are always contained in a property type to ensure that their type is encoded in a surrounding element. Encoded GML is always type/property/type/property. This is also known as the GML "striping" rule. The consequence of this for app-schema mapping files is that ``targetAttributeNode`` must be applied to the property and the type must be set to the XSD property type, not to the type of the contained attribute (``gsml:CGI_TermValuePropertyType`` not ``gsml:CGI_TermValueType``). Because the XPath refers to a property type not the encoded content, ``targetAttributeNode`` appears in a mapping with ``targetAttribute`` and no other elements when using with complex types.
 
+The GML complex features encoder was relaxed to be able to encode nested complex features that are mapped to a complex type that doesn't respect the rule above. The Java configuration property ``encoder.relaxed`` can be set to ``false`` to disable this feature.
```

Comment:
Rather than using the past tense here, perhaps just stick to the present state of the XML encoder. The first sentence then becomes: "The XML encoder will encode nested complex features that are mapped to a complex type that does not respect the GML striping rule."